web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
I'm thinking at first a lot of things will probably be straight CRUD and eventually several things will be more and more complex. That's probably why I favor starting with direct repo calls. 

I do agree the big thing is that we'll want full querying abilities, so the least we that exposed. It's possible that there are other things.

I know with NHibernate a big reason you want to push the session access up in the layers is because you can take advantage of batched queries, eager/lazy loading, but I can't think of anything around CUD that benefits from direct access since the models and the mapping take care of that (which Ecto 2.0 will improve). 

Anyway, I'm cool with your idea of wrapping up the CRUD with the module but allowing the "Read" function to take an ecto query. You addressed my primary concerns in previous comments.

I do want to avoid using primitives everywhere though. Maybe our modules should work off of the structs, so you first gotta query what you want.